features/issues/components/issue-list/issue-list.tsx

```diff
@@ -124,13 +126,7 @@ export function IssueList() {
         <Table>
           <thead>
             <HeaderRow>
-              <HeaderCell>
-                <F.Checkbox
-                  size={F.CheckboxSize.md}
-                  state={F.CheckboxState.partlyChecked}
-                  label="Issue"
-                />
-              </HeaderCell>
```

Comment:
Consider checking this article regarding creating stateful [checkbox](https://blog.logrocket.com/building-custom-checkbox-react/)

Reply:
Will create this into separate feature branch.